docs/Guides/OCP CI Internals/Secrets_Guide.md

```diff
@@ -0,0 +1,123 @@
+
+## Collections
+
+### Create Collection
+Each scenario being tested will have its own collection. This allows us to prevent secret sharing between members of other collections meant to hold secrets for other scenarios.
+
+First go to [selfservice.vault.ci.openshift.org](https://selfservice.vault.ci.openshift.org/secretcollection?ui=true) and login.
+
+If you do not see a collection in the table for the scenario that your testing it either hasn't been created yet or you don't have access to it.
+
+Please reach out on slack at [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) to ask if the collection for the scenario you are testing has already been created. If it has go to the [next section](#get-access-to-collection)
+
+If it hasn't you can create the collection by clicking the `New Collection` button and adding the name. Follow the naming structure of `{product short name}-qe`.
+### Get Access to a Scenario Collection
+If you do not see a collection in the table and you have verified that it exists then you just need to be added by one of the collection owners. Please reach out on slack at [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) and provide details about what and why you would like to be added to a specific collection.
+
+### Get Access to the cspi-qe Collection
+This collection holds team specific information holding things like AWS creds, pull secrets, ..etc. If you need access to this collection reach out on slack at [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) and provide details about why you would like to be added to the cspi-qe collection.
+
+## Vault
+
+### Sign in to Vault
+Now that you are a member of a collection you can login to vault and access that collection.
+
+Go to [vault.ci.openshift.org](https://vault.ci.openshift.org/ui/vault/auth?with=oidc%2F) and click `Sign in with OIDC Provider`.
+
+You'll be redirected to your secrets homepage. 
+`cubbyhole/` is just your users personal space for any secrets you want to store. 
+We care about `kv/` this is where you'll find secret directories that correspond to the collections that you are a member of.
+
+From here you can use the UI to view, create, edit, and delete secrets.
+
+### Secrets in OpenShift CI
+In order to use a secret in Vault in OpenShift CI, the secret must include two key/value pairs:
+- `secretsync/target-name`: The [name](#openshift-ci-secret-name) of the secret in OpenShift CI
+- `secretsync/target-namespace`: The [namespace](#openshift-ci-secret-namespace) of the secret in OpenShift CI
+
+Outside of those two values, any key/value pair desired can be added to the secret for use in OpenShift CI.
```

Comment:
Can we expand what added to the secret means?

This just means adding a key value pair to the same path that you create the `target-name` and `target-namespace` in.

Maybe something like `Outside of those two values, any key/value pair desired can be added to the same path for that secret making them available for use in OpenShift CI.

Reply:
Agreed, updated!